hack/e2e-test.sh

```diff
@@ -33,6 +33,20 @@ if ! which kustomize > /dev/null; then
   popd
 fi
 
+if ! which certbot > /dev/null; then
+  certbot_dir="$(mktemp -d)"
+  export PATH="$PATH:${certbot_dir}"
+  pushd "${certbot_dir}"
+  wget https://dl.eff.org/certbot-auto
```

Comment:
Do we really want to pull unsigned code down like this?

Reply:
We do the same just above for kustomize, I think it's probably ok in e2e test pods. 

It's in epel but I didn't really want to play the "Am I running in EPEL / Fedora / OSX" game in the e2e test script.